serviceUsage.html

```diff
@@ -128,9 +134,13 @@ <h3>Exchange configurations</h3>
 &lt;protocol&gt;.rabbitmq.password:      &lt;password, default for RabbitMQ Server: guest&gt;
 &lt;protocol&gt;.rabbitmq.tls:           &lt;tls version, is optional&gt;
 &lt;protocol&gt;.rabbitmq.exchangeName:  &lt;exchange name, exchange should be already created on RabbitMQ Server&gt;
+&lt;protocol&gt;.rabbitmq.createExchangeIfNotExisting:  &lt;true or false, Exchange will be created when set to true on RabbitMQ Server&gt;
 &lt;protocol&gt;.rabbitmq.domainId:      &lt;domain id, any string&gt;</pre>
 
-        <p>An <a href="https://www.rabbitmq.com/tutorials/tutorial-three-java.html">exchange point</a> must exist before starting the service.</p>
+        <p>An <a href="https://www.rabbitmq.com/tutorials/tutorial-three-java.html">exchange point</a> must exist before starting the service.
```

Comment:
Why repeat the content, can be mentioned only once. Remove this from top.
Change as below:
starting the  service if createExchangeIfNotExisting is set to false or application launch is terminated.
Application can create the exchange by adding the below property in config.properties file.